librosa/sequence.py

```diff
@@ -228,6 +244,9 @@ def dtw(X=None, Y=None, C=None, metric='euclidean', step_sizes_sigma=None,
         else:
             # perform warping path backtracking
             wp = __dtw_backtracking(D_steps, step_sizes_sigma, subseq)
+            if wp[-1] != (0, 0):
+                # TODO: what should we do here?
+                pass
```

Comment:
@fzalkow I added an escape condition to the backtracking loop to avoid array bound exceptions, but we still need to decide what the behavior should be if a full alignment is requested but cannot be constructed.  I'm not totally clear on this TBH.

Reply:
What do you think about giving a warning:
```python
warnings.warn('No valid warping path was computed! The path starts at {} instead of [0 0].'.format(wp[-1]), RuntimeWarning)
```

Reply:
Should it be a warning or an exception?

Reply:
With the inf-check for the accumulated cost matrix, I also don't see when this case occurs... So an exception is maybe more appropriate.

Reply:
I think you're correct.  I haven't been able to construct an example that would trigger this behavior that also somehow ends up with a finite warping path cost.

I'll leave this in as an exception, just in case there is some configuration that could trigger this, but I think it's otherwise safe.